<launch>
    <!--- Run wpb_ai  -->
    <!--- <include file="$(find wpb_ai_bringup)/launch/minimal.launch" /> -->

    <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
        <param name="video_device" value="/dev/video0" />
        <param name="image_width" value="640" />
        <param name="image_height" value="480" />
        <param name="pixel_format" value="yuyv" />
        <param name="camera_frame_id" value="usb_cam" />
        <param name="io_method" value="mmap"/>
        <remap from="/usb_cam/image_raw" to="/kinect2/qhd/image_color"/>

    </node>

    <node pkg="wpb_ai_bringup" name="wpb_ai_yolo_3d" type="wpb_ai_yolo_3d" output="screen"/>

    <node pkg="wpb_ai_bringup" name="yolo_node" type="yolo_node.py"  output="screen"/>
    
    <node name="rqt_image_view" pkg="rqt_image_view" type="rqt_image_view" respawn="false" output="screen">
        <param name="autosize" value="true" />
    </node>

</launch>